### PR TITLE
feat: exclude DEBUG methods and Token contract from gas report

### PR DESCRIPTION
**Pull request by unparalleled-js**
Requires this PR: https://github.com/ApeWorX/ape/pull/1111

### CODE REVIEW

**Review by fubuloubu — APPROVED** [review]

File: ape-config.yaml [hunk 1]
```diff
@@ -10,3 +10,8 @@ dependencies:
 solidity:
   import_remapping:
     - "@OpenZeppelin=OpenZeppelin/4.7.3"
+
+test:
+  gas:
+    exclude:
```
Comment (fubuloubu) [inline ape-config.yaml:16]:
```suggestion
    exclude:
      - contract: Token
```

**Review by fubuloubu — APPROVED** [review]
Needs passing CI once ape PR is merged and released

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-eth-ape>=0.5.1
+eth-ape>=0.5.2
```
Comment (unparalleled-js) [inline requirements.txt:1]: Commit after release of eth-ape

```suggestion
eth-ape>=0.5.4
```

**Comment by unparalleled-js** [comment]
I am guessing there is an issue with our GH action's caching of eth-ape.
My suspicion is that it caches Ape under a version greater than its actual version

**Review by fubuloubu — APPROVED** [review]

File: requirements.txt (hunk 2 above)
Comment (NotPeopling2day) [reply requirements.txt:1]: Try adding this to re-trigger the build and see if that fixes the Pydantic validation

**Pull request merged**